### ISSUE TITLE
Feature Request: Lower Burst Rate, Longer Duration

### ISSUE THREAD

**Issue by revivaljester** [PkmX/lcamera#142]
It's known that the Nexus 5 can shoot 30 frames in a second before the buffer fills up, but is it possible to have a lower burst rate so that one can maintain burst shooting for much longer? For example, iPhones 5S and newer iterations shoot at a burst rate of 10fps which seems to strike a balance between speed and bandwidth bottlenecks.

**Comment by kagwind** [comment]
I think this is a good idea.

**Comment by Fishywalrus** [comment]
Agreed.. an option for how many FPS (eg. between 2 and the full 30fps) would be incredibly useful..
